### DISCUSSION TITLE
publish table index blade

### DISCUSSION THREAD

**Discussion by juangdnar** [filamentphp/filament#9656] [Help] [answered]
It is possible to publish the index file of the tables found in \vendor\filament\tables\resources\views\index.blade.php in order to reorganize some elements, such as search field filters

**Comment by leandrocfe** [comment, marked as answer]
This [can be dangerous](https://filamentphp.com/community/danharrin-publishing-views-in-laravel), but you can add this view `resources/views/vendor/filament-tables/index.blade.php`, copy the original content and edit what you need.

**Reply by juangdnar** [reply]
it works!!! thanks a lot

**Reply by ahmadthedev** [reply]
Saved my time!
